cmake/FindVecGeom.cmake

```diff
@@ -21,17 +21,19 @@ cmake_policy(POP)
 include(FindPackageHandleStandardArgs)
 find_package_handle_standard_args(VecGeom CONFIG_MODE)
 
-if(VecGeom_FOUND AND TARGET VecGeom::vecgeomcuda)
+if(VecGeom_FOUND AND (TARGET VecGeom::vecgeomcuda OR TARGET VecGeom::vecgeom_final))
```

Comment:
It would be useful to have version ranges for what vecgeom has the different behavior: if we stop supporting 1.x for example some of this code can go.

Reply:
With all the streams, it might be a mess :( [ v1, v2, surface ] ) ... I will add a comment with the versions number I can find.

Reply:
I think only 1.x and 2.0+ (aka beyond current master) need to be worried about: it's just a comment after all.